crates/bevy_pbr/src/wireframe.rs

```diff
@@ -194,7 +194,7 @@ fn apply_global_wireframe_material(
     }
 }
 
-#[derive(Default, AsBindGroup, TypeUuid, TypePath, Debug, Clone, Asset)]
+#[derive(Default, AsBindGroup, TypePath, Debug, Clone, Asset)]
 #[uuid = "9e694f70-9963-4418-8bc1-3474c66b13b8"]
```

Comment:
This uuid should also be removed correct?

Reply:
yup.